models.py

```diff
@@ -2,67 +2,104 @@
 from uuid import uuid4
 
 import requests
+from datetime import datetime
 from kytos.core import log
-from kytos.core.helpers import now
+from kytos.core.helpers import now, get_time
 from kytos.core.interface import UNI
 from napps.kytos.mef_eline import settings
 
 
 class EVC:
     """Class that represents a E-Line Virtual Connection."""
 
-    def __init__(self, uni_a, uni_z, name, start_date=None, end_date=None,
-                 bandwidth=None, primary_links=None, backup_links=None,
-                 dynamic_backup_path=None, creation_time=None):
+    def __init__(self, **kwargs):
         """Create an EVC instance with the provided parameters.
 
-        Do some basic validations to attributes.
+        Args:
+            id(str): EVC identifier. Whether it's None an ID will be genereted.
+            name: represents an EVC name.(Required)
+            uni_a (UNI): Endpoint A for User Network Interface.(Required)
+            uni_z (UNI): Endpoint Z for User Network Interface.(Required)
+            start_date(datetime|str): Date when the EVC was registred.
+                                      Default is now().
+            end_date(datetime|str): Final date that the EVC will be fineshed.
+                                    Default is None.
+            bandwidth(int): Bandwidth used by EVC instance. Default is 0.
+            primary_links(list): Primary links used by evc. Default is []
+            backup_links(list): Backups links used by evc. Default is []
+            dynamic_backup_path(list): None
```

Comment:
@macartur based on EP012.rst, dynamic_backup_path is supposed to be bool. It's just a flag to enable if the backup paths should be computed dynamically, in case other backup paths are available.